ament_clang_tidy/ament_clang_tidy/main.py

```diff
@@ -214,8 +214,14 @@ def start_subprocess(full_cmd):
     for filename in files:
         report[filename] = []
 
-    error_re = re.compile('(/.*?\\.(?:%s)):(\\d+):(\\d+): (?:warning:|error:)' %
-                          '|'.join(extensions))
+    # Regex explanation:
+    #   \s*\^?                   : Capture many spaces, then an optional `^`. clang_tidy output isn't great.
+    #   (\/home\/^[^:]*)         : Group capture. Everything from `/home/` up until a `:`.
```

Comment:
```suggestion
    #   (/home/^[^:]*)         : Group capture. Everything from `/home/` up until a `:`.
```

I don't think that `/` is a metacharacter outside of tools like `sed` when it is used as a pattern delimiter.

Reply:
Good catch! Will drop this in a commit once I test to make sure functionality is the same.

Reply:
Resolved in 5d5e2eda